docs/miscellaneous.rst

```diff
@@ -168,15 +168,14 @@ These steps are applied to each basic block and the newly generated code is used
     else
       return 1;
 
-is simplified to code which can also be compiled from
+Still simplifies to code which you can compile even though the instructions contained 
```

Comment:
This sentence does not have a subject.

Reply:
@chriseth I don't actually agree in this case, nor do I personally like the style of splitting a sentence with a code example of a couple of lines. For now, I will revert this to the older style, and I shortened the split sentence to make it flow better.

I think we need to create a documentation style guide to help me and other contributors know what style contributions should follow, and then ideally use that to check contributions. What do you think?

Reply:
Sounds like a good idea!

I'm just saying: Either split the sentence into half-sentences with code being objects or you full sentences. I agree that the latter should be preferred.

Reply:
@chriseth Awesome, in this case I couldn't think of a way to resolve it ideally. But maybe I will in a future PR. Style guide noted.